lib/methods/template.js

```diff
@@ -75,7 +75,12 @@ const splitByWhitespaces = (template, rawTemplate) => {
 			templateStart = templateIndex + 1;
 		} else if (rawCharacter === '\\') {
 			const nextRawCharacter = rawTemplate[rawIndex + 1];
-			if (nextRawCharacter === 'u' && rawTemplate[rawIndex + 2] === '{') {
+			if (nextRawCharacter === '\n') {
```

Comment:
This may need to also handle `\r\n` but I'm not sure how to write a test for that.

Reply:
No because escaped newlines inside template strings always use Unix newlines (inside `template.raw`), even on Windows.